### PR TITLE
Modified spanish translation...

### PR DESCRIPTION
**Pull request by jrizza**
I have modified certain words and phrases, some of them I guess should stay in english....

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -232,12 +232,12 @@ es:
           <p><em>Nota: Nota: Puede utilizar un archivo exportado útiles desde otro sistema (Pivotal Tracker).</em></p>
 
   users:
-    You are not authorized to perform this action: "Usted no está autorizado para realizar esta acción."
+    You are not authorized to perform this action: "Usted de otro  autorizado para realizar esta acción."
```
Comment (daniloisr) [inline config/locales/es.yml]: This change seems to be wrong. Can you check it?

Comment (jrizza) [inline config/locales/es.yml]: Missed that, sorry, I'm fixing it...

**Comment by jrizza** [comment]
Please check now...

**Comment by daniloisr** [comment]
Thanks!

**Pull request merged**